etc/openqa/openqa.ini

```diff
@@ -78,6 +78,8 @@
 ## Causes jobs reported as incomplete by the worker to be cloned automatically when the
 ## reason matches; set to 0 to disable
 #auto_clone_regex = ^(cache failure: |terminated prematurely: |api failure: Failed to register .* 503|backend died: .*VNC.*(timeout|timed out|refused)|QEMU terminated: Failed to allocate KVM HPT of order 25.* Cannot allocate memory)
+## How many times should a job be restarted
+#auto_clone_limit = 20
```

Comment:
As your new approach is now not looking into the actual reason to determine if we should limit I suggest to consider a new variable name if you have a good idea. And also I suggest to increase the default value then, let's say 50?

Reply:
nevermind, you are still looking for any auto_clone regex match so all good